### PR TITLE
Add more dynamic example to the choose command usage

### PR DESCRIPTION
**Pull request by BlackCoyote**

### CODE REVIEW

File: src/main/java/net/aufdemrand/denizencore/scripts/commands/CommandRegistry.java [hunk 1]
```diff
@@ -158,6 +158,18 @@ public void registerCoreCommands() {
         //   - default:
         //     - debug log "Success!"
 
+        // @usage
```
Comment (mcmonkey4eva) [inline src/main/java/net/aufdemrand/denizencore/scripts/commands/CommandRegistry.java:161]: Capital `U`

Comment (BlackCoyote) [reply src/main/java/net/aufdemrand/denizencore/scripts/commands/CommandRegistry.java:161]: pls dont make me do all this again :(

**Comment by mcmonkey4eva** [comment]
I'm not sure if this is valid for DCore, to have DBukkit-specific tags...

**Comment by BlackCoyote** [comment]
gjhgkjhgdlkjhg fine

**Pull request closed without merging**